Update create-a-phone-system-call-queue.md (#3846)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/create-a-phone-system-call-queue.md

**Routing method** You can choose either **Attendant**, **Serial**, or **Round Robin** as the distribution method. All new and existing call queues have attendant routing selected by default. When attendant routing is used, the first call in the queue rings all call agents at the same time. The first call agent to pick up the call gets the call.

- **Attendant routing** causes the first call in the queue to ring all call agents at the same time. The first call agent to pick up the call gets the call.
- **Serial routing** incoming calls ring all call agents one by one, from the beginning of the call agent list. Agents can't be ordered within the call agent list. If an agent dismisses or does not pick up a call, the call will ring the next agent and will try all agents until it is picked up or times out.
  > [!NOTE]

>   > With Serial routing, for agents who are **Offline** or have set their presence to **Do not Disturb**, the call will be routed to those users and fail to connect unavailable user, the routing to the next agent in the agent list. This is not the case if the agent has **opted out** of getting calls from the call queue. To reduce the time interval the call routes to next agent in line, Agent Alert time can be decreased.

Reviewer: ```suggestion
  > With Serial routing, for agents who are **Offline** or have set their presence to **Do not Disturb**, the call will still be routed to those agents and and when failing to connect to the unavailable agents, will be routed to the next agent in the agent list. This is not the case if the agent has **opted out** of getting calls from the call queue. To reduce the time interval before the call routes to the next agent in line, Agent Alert time can be decreased.
```

Reviewer 2: Exactly! Thanks for putting it that clear 😊

Reviewer 2: Hi @JohanFreelancer9 , do you have any idea when these changes will be applied to the official documentation ? Thank you !

Reviewer 2: Hi @JohanFreelancer9 . It seems there is a grammar mistake here, as it does not make much sence:
_the call will be routed to those users and fail to connect unavailable user, the routing to the next agent in the agent list_
maybe that is more clear:
the call will be routed to those users and fail to connect **an** unavailable user, **after the specified agent alert time the call will be routed** to the next agent in the agent list.

Reviewer: @scanum It seems that my copy review was not been merged.
